dse/Dockerfile-dse6.8.jdk11

```diff
@@ -191,6 +196,27 @@ RUN if ! grep -qxF "JVM_OPTS=\"\$JVM_OPTS -javaagent:${MGMT_AGENT_JAR}\"" < ${CA
       echo "" >> ${CASSANDRA_CONF}/cassandra-env.sh && \
       echo "JVM_OPTS=\"\$JVM_OPTS -javaagent:${MGMT_AGENT_JAR}\"" >> ${CASSANDRA_CONF}/cassandra-env.sh ; \
   fi
+# Add MCAC check to cassandra-env.sh
+RUN if ! grep -qxf "JVM_OPTS=\"\$JVM_OPTS -Dinsights.default_mode=disabled\"" < ${CASSANDRA_CONF}/cassandra-env.sh ; then \
+    echo "" >> ${CASSANDRA_CONF}/cassandra-env.sh && \
+    echo "if [ ! -z \"\$MGMT_API_DISABLE_MCAC\" ]; then" >> ${CASSANDRA_CONF}/cassandra-env.sh && \
+    echo "  JVM_OPTS=\"\$JVM_OPTS -Dinsights.default_mode=disabled\"" >> ${CASSANDRA_CONF}/cassandra-env.sh && \
+    echo "fi" >> ${CASSANDRA_CONF}/cassandra-env.sh ; \
+  fi
+# link /config/jmxremote.password if it exists and there isn't one in CASSANDRA_CONF
+RUN if [ -e "/config/jmxremote.password" ] && [ ! -e "$CASSANDRA_CONF/jmxremote.password" ]; then \
+    ln -s /config/jmxremote.password $CASSANDRA_CONF/jmxremote.password ; \
+  fi
```

Comment:
@burmanm Will this ever work?
The `/config` directory would be a volume mount in the case of cass-operator, right? So when the image is built, the Dockerfile will never see a `/config/jmxremote.password` file, right. I don't think moving this bit in here will work the same way it does in the entrypoint script, which isn't executed until after volume mounts are established.

Reply:
I'm not sure if we support remote JMX in DSE at all? So it wouldn't make any difference even if we removed this file entirely.

Reply:
Ok, I'll do that.